content/en/docs/_index.md

```diff
@@ -5,14 +5,18 @@ menu: { main: { weight: 10 } }
 aliases: [/docs/workshop/*]
 ---
 
-OpenTelemetry, also known as OTel for short, is a vendor-neutral open source
+OpenTelemetry, also known as OTel, is a vendor-neutral, open source
 [Observability](concepts/observability-primer/#what-is-observability) framework
 for instrumenting, generating, collecting, and exporting telemetry data such as
 [traces](/docs/concepts/signals/traces/),
-[metrics](/docs/concepts/signals/metrics/),
-[logs](/docs/concepts/signals/logs/). As an industry-standard, it is
-[supported by 40+ observability vendors](/ecosystem/vendors/), integrated by
-many [libraries, services and apps](/ecosystem/integrations) and adopted by a
-[number of end-users](/ecosystem/adopters).
+[metrics](/docs/concepts/signals/metrics/), and
+[logs](/docs/concepts/signals/logs/).
+
+As an industry-standard, OpenTelemetry is
+[supported by more than 40 observability vendors](/ecosystem/vendors/),
+integrated by many [libraries, services and apps](/ecosystem/integrations), and
+adopted by [numerous end users](/ecosystem/adopters).
+
+The following diagram shows the high-level OpenTelemetry architecture:
```

Comment:
IMHO, this **isn't** an architecture diagram of OpenTelemetry. I'm not sure what to call it atm, and I don't think that it's worth spending too much time figuring that out, so I'd pull out the sentence:

```suggestion
```

_Maybe_ we can call it a high-level and abstract systems architecture diagram illustrating the flow of telemetry data between the OpenTelemetry Collector and instrumented components.

Reply:
Let's remove it for now, though it'd be great to introduce all our diagrams somehow.